ppr-ui/src/components/mhrRegistration/HomeLocation/HomeCivicAddress.vue

```diff
@@ -71,14 +74,56 @@ import { defineComponent, reactive, toRefs, watch } from '@vue/composition-api'
 import { CivicAddressSchema } from '@/schemas/civic-address'
 import { useActions, useGetters } from 'vuex-composition-helpers'
 import { useMhrValidations } from '@/composables'
+import {
+  baseRules,
+  useAddress,
+  useAddressComplete,
+  useCountryRegions,
+  useCountriesProvinces,
+  useBaseValidations,
+  spaceRules
+} from '@/composables/address/factories'
+import { AddressIF } from '@/interfaces'
+import { SchemaIF } from '@/composables/address/interfaces'
 /* eslint-enable no-unused-vars */
 export default defineComponent({
   name: 'HomeCivicAddress',
   components: {},
   props: {
+    value: {
+      type: Object as () => AddressIF,
+      default: () => ({
+        street: '',
+        streetAdditional: '',
+        city: '',
+        region: '',
+        postalCode: '',
+        country: '',
+        deliveryInstructions: ''
+      })
+    },
     validate: {
       type: Boolean,
       default: false
+    },
+    editing: {
+      type: Boolean,
+      default: false
+    },
```

Comment:
Are you using this prop?

Reply:
Removed.  Not being used in this component.  Copied from BaseAddress..